rama-net/src/fingerprint/peet/tls.rs

```diff
@@ -0,0 +1,625 @@
+//! PeetPrint implementation for Rama (in Rust).
+//!
+//! PeetPrint is inspired by the custom fingerprint algorithm from TrackMe.
+//! See license information below:
+//!
+//! > Original work from:
+//! >   https://github.com/pagpeter/TrackMe
+//! >
+//! > Licensed under GPLv3.
+//! > See https://github.com/plabayo/rama/blob/peet-print/docs/thirdparty/licenses/pagpeter-trackme for license details.
```

Comment:
```suggestion
//! > See <https://github.com/plabayo/rama/blob/main/docs/thirdparty/licenses/pagpeter-trackme> for license details.
```

You can already prepare the correct link